plugins/woo-ai/package.json

```diff
@@ -39,9 +39,11 @@
 		"wireit": "0.14.3"
 	},
 	"scripts": {
-		"build": "pnpm build:admin && pnpm uglify",
+		"build": "pnpm --if-present --workspace-concurrency=Infinity --stream --filter=\"$npm_package_name...\" '/^build:project:.*$/'",
 		"build:admin": "wp-scripts build",
```

Comment:
We should probably just remove this entirely.

Reply:
I was highlighting the bottom `build:admin` line. We should remove `build:admin` and probably add the uglification step to the `wireit` build command.

Reply:
I might be missing something here, but the `uglify` script is defined as:
```
"uglify": "rm -f $npm_package_assets_js_min && for f in $npm_package_assets_js_js; do file=${f%.js}; node_modules/.bin/uglifyjs $f -c -m > $file.min.js; done",
```

However, there's no config in the `package.json` for `assets.js.min` or `assets.js.js`. Is this step actually doing anything the way it's configured currently? cc @retrofox

Reply:
Honestly since `webpack` is configured here we can just remove the uglification script entirely. Not only does the current command not actually do anything this is behavior that should just be configured in `webpack.config.js` if we want it.